zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/zeppelinhub/websocket/ZeppelinClient.java

```diff
@@ -65,6 +69,12 @@
   private Authentication authModule;
   private static final int MIN = 60;
 
+  private static final Set<String> actionable = new  HashSet<String>(Arrays.asList(
```

Comment:
I am not sure to understand what this list is made for?
Because i feel like its a list of OP event that will be forwarded to zeppelinHub but (if i am right) looks like this list is incomplete.